pkg/webui/console/store/reducers/pubsubs.js

```diff
@@ -61,7 +61,25 @@ const pubsubs = function(state = defaultState, { type, payload }) {
         ...state,
         entities: {
           ...state.entities,
-          [pubsubId]: merge({}, state.entities[pubsubId], payload),
+          [pubsubId]: mergeWith(
```

Comment:
Why don't we just use the whole return value as new value without merging, since the backend will return the whole updated object anyway?